cmd/crank/beta/validate/cache.go

```diff
@@ -32,7 +32,7 @@ type Cache interface {
 	Store(schemas [][]byte, path string) error
 	Flush() error
 	Init() error
-	Load() ([]*unstructured.Unstructured, error)
+	Load(path string) ([]*unstructured.Unstructured, error)
```

Comment:
nit: the interface uses `path`, but the implementation uses `image` - consolidate on `image` i'd say

Reply:
oh right! Going to fix it.